docs/pages/includes/database-access/guides.mdx

```diff
@@ -25,6 +25,7 @@
 - [Self-hosted Redis](../../database-access/guides/redis.mdx): Connect self-hosted Redis.
 - [Self-Hosted Cassandra & ScyllaDB](../../database-access/guides/cassandra-self-hosted.mdx): Connect self-hosted Cassandra or ScyllaDB.
 - [Self-Hosted Oracle](../../database-access/guides/oracle-self-hosted.mdx): Connect self-hosted Oracle database.
```

Comment:
not related to click house but should Oracle here has a preview tag?

Reply:
Yea, It looks like the Preview tags is missing I will address this in a separate PR.